src/Symfony/Component/Serializer/Tests/Normalizer/NumberNormalizerTest.php

```diff
@@ -51,20 +52,40 @@ public static function supportsNormalizationProvider(): iterable
     }
 
     /**
-     * @dataProvider normalizeGoodValueProvider
+     * @requires extension bcmath
+     *
+     * @dataProvider normalizeGoodBcMathNumberValueProvider
      */
-    public function testNormalize(mixed $data, mixed $expected)
+    public function testNormalizeBcMathNumber(mixed $data, mixed $expected)
```

Comment:
The arguments types can be narrowed now it seems? Same for other tests using data providers

Reply:
indeed, I changed them where possible